src/Apple/composer.json

```diff
@@ -36,7 +36,7 @@
         "ext-openssl": "*",
         "firebase/php-jwt": "^5.2",
         "lcobucci/jwt": "^4.0",
-        "socialiteproviders/manager": "~4.0"
+        "socialiteproviders/manager": "^4.0"
```

Comment:
`~4.0` allows 4.1. Can you post the output of composer update, something else is causing the conflict.